internal/storage/bucket_handle_test.go

```diff
@@ -256,3 +256,18 @@ func (t *BucketHandleTest) TestCreateObjectMethodWhenGivenGenerationObjectNotExi
 	AssertEq(nil, obj)
 	AssertTrue(strings.Contains(err.Error(), "Error 412: Precondition failed"))
 }
+
+func (t *BucketHandleTest) TestListObjectMethod() {
+	obj, err := t.bucketHandle.ListObjects(context.Background(),
+		&gcs.ListObjectsRequest{
+			Prefix:                   "gcsfuse/default.txt",
```

Comment:
can you add more test cases to cover the following scenarios:
1. testing prefix parameter - > with data present and not present
2. testing delimiter paramters -> for different type of results
3. including trailing delimiter param
4. projection value

Reply:
fixed